### PR TITLE
Add a map view to Main/Map using OpenLayer API

### PR DESCRIPTION
**Pull request by narekacy**
Take a look to our new map feature.
If everything is ok, just approve it.

### CODE REVIEW

File: client/src/components/map/openlayers/OLMap.js [hunk 1]
```diff
@@ -0,0 +1,26 @@
+import React, {Component} from "react";
+import "./styles.css";
+
+import {Map, layer, Layers} from "react-openlayers";
+
+class OLMap extends Component {
+  render() {
+    return (
+      <div className="Map">
```
Comment (VanderbiGit) [inline client/src/components/map/openlayers/OLMap.js]: We use react-bootstrap. You need Container.

File: client/src/components/map/openlayers/styles.css [hunk 2]
```diff
@@ -0,0 +1,3 @@
+.openlayers-map {
```
Comment (VanderbiGit) [inline client/src/components/map/openlayers/styles.css]: .css must be  in src/style/your_css
!important its bad practice

File: client/src/pages/Map.js [hunk 3]
```diff
@@ -1,12 +1,13 @@
 import React, { Component } from 'react';
+import OLMap1 from '../components/map/openlayers/OLMap'
 
 
 class Map extends Component {
     render() {
         return (
-            <h1>Map Page</h1>
+            <h1><OLMap1/></h1>
```
Comment (VanderbiGit) [inline client/src/pages/Map.js]: h1 not need

**Review by vitaliimak — COMMENTED** [review]
All functionality works well. But as we decided to use kebab-case in file naming you could rename file OLMap.js to ol-map.js

File: client/src/components/map/openlayers/OLMap.js (hunk 1 above)
Comment (narekacy) [reply client/src/components/map/openlayers/OLMap.js]: Thank you for hint. Already fixed. Check it, please.

File: client/src/components/map/openlayers/styles.css (hunk 2 above)
Comment (narekacy) [reply client/src/components/map/openlayers/styles.css]: Thank you for hint. Already fixed. Check it, please.

File: client/src/pages/Map.js (hunk 3 above)
Comment (narekacy) [reply client/src/pages/Map.js]: Just missed this single piece of code :)

**Review by vitaliimak — APPROVED** [review]

**Review by VanderbiGit — APPROVED** [review]

**Review by r-yevchuk — APPROVED** [review]

**Pull request merged**